modules/cnf-checking-numa-aware-scheduler-logs.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-checking-numa-aware-scheduler-logs_{context}"]
+= Checking the NUMA-aware scheduler logs
+
+Troubleshoot problems with the NUMA-aware scheduler by looking at the logs. If required, you can increase the scheduler log level by modifying the `spec.logLevel` field of the `NUMAResourcesScheduler` resource. Acceptable values are `Normal`, `Debug`, and `Trace`, with `Trace` being the most verbose option.
+
+[NOTE]
+====
+To change the log level of the secondary scheduler, delete the running scheduler resource and re-deploy it with the changed log level. The scheduler is unavailable for scheduling new workloads during this downtime.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Delete the currently running `NUMAResourcesScheduler` resource:
+
+.. Get the active `NUMAResourcesScheduler`:
++
+[source,terminal]
+----
+$ oc get NUMAResourcesScheduler
+----
++
+.Example output
+[source,terminal]
+----
+NAME                     AGE
+numaresourcesscheduler   90m
+----
+
+.. Delete the secondary scheduler resource:
++
+[source,terminal]
+----
+$ oc delete NUMAResourcesScheduler numaresourcesscheduler
+----
++
+.Example output
+[source,terminal]
+----
+numaresourcesscheduler.nodetopology.openshift.io "numaresourcesscheduler" deleted
+----
+
+. Save the following YAML in the file `nro-scheduler-debug.yaml`. This example changes the log level to `Debug`:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1alpha1
+kind: NUMAResourcesScheduler
+metadata:
+  name: numaresourcesscheduler
+spec:
+  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel8:v4.10"
+  logLevel: Debug
+----
+
+. Create the updated `Debug` logging `NUMAResourcesScheduler` resource:
++
+[source,terminal]
+----
+$ oc create -f nro-scheduler-debug.yaml
+----
++
+.Example output
+[source,terminal]
+----
+numaresourcesscheduler.nodetopology.openshift.io/numaresourcesscheduler created
+----
+
+.Verification steps
+
+. Check that the NUMA-aware scheduler was successfully deployed:
+
+.. Check that the CRD is created succesfully:
```

Comment:
In-house guidelines: "If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to 'run' or 'enter' the command."

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Apply throughout PR.

Reply:
I disagree with having to use this construction for every step. It goes against basic minimalism IMO and makes the content more clunky for the reader. Maybe one for further discussion. Also: https://coreos.slack.com/archives/C025XCRRH28/p1655819386472169?thread_ts=1654869028.203199&cid=C025XCRRH28

Reply:
I made these changes as required by ISG.